core/renderer/backend/opengl/RenderTargetGL.cpp

```diff
@@ -46,12 +46,11 @@ void RenderTargetGL::unbindFrameBuffer() const
 void RenderTargetGL::update() const {
     if (!_dirty) return;
     if(!_defaultRenderTarget) {
-        if(bitmask::any(_flags, TargetBufferFlags::COLOR_ALL))
         { // color attachments
             GLenum bufs[MAX_COLOR_ATTCHMENT] = {GL_NONE};
             for (size_t i = 0; i < MAX_COLOR_ATTCHMENT; ++i)
             {
-                if (bitmask::any(_flags, getMRTColorFlag(i)))
+                if (_color[i])
```

Comment:
due to _flags was removed, need update color attachment always

Reply:
@solan-solan remove the condition check `_color[i]` can solve above GL errors

Reply:
> @solan-solan remove the condition check `_color[i]` can solve above GL errors

@halx99 I believe that it helps